frontend/js/recordings.js

```diff
@@ -3,7 +3,32 @@ import SimpleTemplate from "./simple-template.js";
 // the specific URL for a given wordform (refactored from previous commits).
 // TODO: should come from config.
 const BASE_URL = "https://speech-db.altlab.app";
-const BULK_API_URL = `${BASE_URL}/api/bulk_search`;
+const LANGUAGE_CODE = getLanguageCodeFromLocation();
+const BULK_API_URL = `${BASE_URL}/${LANGUAGE_CODE}/api/bulk_search`;
+
+
+function getLanguageCodeFromLocation() {
+    const location = window.location.toString();
```

Comment:
Ha neat, I didn't know `location` stringifies that way. I usually go with `window.location.href`.